quality_control_mrp/views/qc_inspection_view.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="qc_inspection_form_view_production">
+            <field name="name">qc.inspection.form.view.production</field>
+            <field name="model">qc.inspection</field>
+            <field name="inherit_id" ref="quality_control.qc_inspection_form_view" />
```

Comment:
You should also protect this view with the mrp base group for those who don't have access to mrp